velox/exec/CMakeLists.txt

```diff
@@ -109,6 +109,22 @@ velox_link_libraries(
   velox_arrow_bridge
   velox_common_compression)
 
+velox_add_library(velox_cursor Cursor.cpp)
```

Comment:
What is the failure that requires this move?

Reply:
https://github.com/facebookincubator/velox/actions/runs/11947025017/job/33302337531#step:6:3179
LocalRunner uses this, when static linking it somehow transitively found it's way in but shared linking is more stringent. The fix might be to refactor the use out of LocalRunner but that's work for someone else^^